moveit_ros/visualization/motion_planning_rviz_plugin/include/moveit/motion_planning_rviz_plugin/motion_planning_frame.h

```diff
@@ -100,6 +100,7 @@ class MotionPlanningFrame : public QWidget
   Q_OBJECT
 
 public:
+  MotionPlanningFrame(const MotionPlanningFrame&) = delete;
```

Comment:
Ah, I see this now makes sure it can't get copied. The move constructor is also automatically deleted in this scenario if I'm not mistaken, so that makes the delete in the destructor always safe :)